cfme/test_requirements.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+alert = pytest.mark.requirement("alert")
+auth = pytest.mark.requirement("auth")
+automate = pytest.mark.requirement("automate")
+black = pytest.mark.requirement("black")
+c_and_u = pytest.mark.requirement("c_and_u")
```

Comment:
Secondly, I don't like the fact that a test will have a decorator that says just
```python
@alert
def test():
```

I would prefer to see either

```python
@pytest.mark.requirement(auth)
def test():
```

or

```python
@REQUIREMENT.auth
def test():
```

Thoughts?